webdriver/tests/bidi/browsing_context/locate_nodes/invalid.py

```diff
@@ -76,7 +76,7 @@ async def test_params_locator_accessability_value_invalid_type(
     ("xpath", ""),
     ("innerText", ""),
     ("accessibility", {}),
-    ("context", {})
+    ("context", {"context": ""})
```

Comment:
Here we should have a dedicated test as well for invalid types similar to the accessibility locator. 

Note that the accessibility value the line above is wrong as well and is actually an invalid type test.